src/org/zalando/stups/essentials/api.clj

```diff
@@ -61,7 +84,9 @@
 (defn read-resource-types
   "Provides a list of all resource types"
   [_ request db]
-  (u/require-internal-user request)
+  (if (:tokeninfo request)
+    (u/require-realms #{"services" "employees"} request)
+    (log/warn "Could not validate authorization due to missing tokeninfo. Set HTTP_TOKENINFO_URL to enable full validation"))
```

Comment:
do we really need this warn log on eahc request? I can imagine this is annoying for someone who doesn't want these extra rules. The warn log appears already once when the app is started without HTTP_TOKENINFO_URL